src/test/java/com/kiwigrid/helm/maven/plugin/UploadMojoTest.java

```diff
@@ -28,6 +28,7 @@
 @MojoProperty(name = "helmDownloadUrl", value = "https://kubernetes-helm.storage.googleapis.com/helm-v2.9.1-linux-amd64.tar.gz")
 @MojoProperty(name = "chartDirectory", value = "junit-helm")
 @MojoProperty(name = "chartVersion", value = "0.0.1")
+@MojoProperty(name = "appVersion", value = "1.0")
```

Comment:
If appVersion becomes optional, this will be no longer necessary.

Reply:
Fixed